lens/sqlrepo/tstracker/chainbs.go

```diff
@@ -0,0 +1,118 @@
+package tstracker
```

Comment:
@willscott I believe this entire file ( `tstracker/chainbs.go` ) can be deleted now

Reply:
must be the merge conflict.